docsite/source/configuration.html.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Configuration
+layout: gem-single
+name: dry-operation
+---
+
+By default, dry-operation automatically wraps the `#call` method of your operations with failure tracking and [error handling](docs::error-handling). This is what allows you to use `#step` directly in your `#call` method. Without this, it would be necessary to wrap all steps in a `steps do ... end` block.
```

Comment:
```suggestion
By default, dry-operation automatically wraps the `#call` method of your operations with failure tracking and [error handling](docs::error-handling). This is what allows you to use `#step` directly in your `#call` method.
```

What do you think about removing this sentence? It's the first time the reader will encounter anything to do with `steps do ... end`, but we don't actually explain it right away, so IMO it's potentially confusing.

If we remove this sentence, I think the documentation still flows well, and the reader will properly learn about `steps do ... end` in the "Disabling automatic wrapping" section just a bit further down the page.

What do you reckon?

Reply:
Make sense 👍 Done